api/models.py

```diff
@@ -14,7 +16,7 @@ class UserProfile(models.Model):
 	profile_picture = models.ImageField(upload_to=settings.MEDIA_UPLOAD_FOLDER, blank=True)  
 	website = models.URLField(blank=True)
 	bio = models.TextField(max_length=2000, blank=True)
-	activity_status = models.CharField(max_length=50, choices=ACTIVITY_LEVEL_CHOICES, default='ACTIVE')
+	activity_status = models.CharField(max_length=50, choices=ACTIVITY_LEVEL_CHOICES, default='KITTEN')
```

Comment:
Re: default=kitten - what about somebody, who signs up and is not a member of CodeCatz?